xlive/XLive/XUser/XUser.cpp

```diff
@@ -55,6 +55,34 @@ XUSER_SIGNIN_INFO* UserGetSignInInfo(DWORD dwUserIndex)
 	return &usersSignInInfo[dwUserIndex];
 }
 
+void XUserSetupGuests(XUID primary_xuid,bool online)
+{
+	for (int dwUserIndex = 1; dwUserIndex < 4; dwUserIndex++)
```

Comment:
uint32 instead of int

Reply:
Not going to use uint32 or other custom defined types of cseries inside an XLIVE api.
Non of the functions in that file use it either .

Reply:
you should use a DWORD instead for `dwUserIndex` since there's a type mismatch when it's used to set `dwGuestNumber`